tvtk/pyface/ui/qt4/QVTKRenderWindowInteractor.py

```diff
@@ -174,19 +89,17 @@ def __init__(self, parent=None, wflags=Qt.WindowFlags(), **kw):
             rw = None
 
         # create qt-level widget
-        QWidget.__init__(self, parent, wflags|Qt.MSWindowsOwnDC)
+        QWidget.__init__(self, parent, wflags | Qt.MSWindowsOwnDC)
```

Comment:
Maybe a comment as to why we are not calling the parent's __init__ would be good.